### PR TITLE
Update changelog for 1.6.0 release

### PR DESCRIPTION
**Pull request by swissspidy**
Updates the changelog in `readme.txt` (used on WordPress.org) for the planned 1.6.0 release.

See #7194

### CODE REVIEW

File: readme.txt [hunk 1]
```diff
@@ -108,6 +108,16 @@ Web Stories are powered by [AMP](https://amp.dev/), which adds some restrictions
 
 For the plugin's full changelog, please see [the Releases page on GitHub](https://github.com/google/web-stories-wp/releases).
 
+= 1.6.0 =
+
+**Release Date:** April 13, 2021.
+
+* New feature: automated video optimization during upload.
+* New feature: canvas zoom levels, to allow for pixel-perfect designs.
```
Comment (dreamofabear) [inline readme.txt]:
```suggestion
* New feature: canvas zoom for pixel-perfect designs.
```

Nit: comma splice.

File: readme.txt [hunk 2]
```diff
@@ -108,6 +108,16 @@ Web Stories are powered by [AMP](https://amp.dev/), which adds some restrictions
 
 For the plugin's full changelog, please see [the Releases page on GitHub](https://github.com/google/web-stories-wp/releases).
 
+= 1.6.0 =
+
+**Release Date:** April 13, 2021.
+
+* New feature: automated video optimization during upload.
```
Comment (dreamofabear) [inline readme.txt]: Should we mention support for .mov files?

File: readme.txt [hunk 3]
```diff
@@ -174,6 +184,10 @@ For the plugin's full changelog, please see [the Releases page on GitHub](https:
 
 == Upgrade Notice ==
 
+= 1.6.0 =
+
+New built-in video optimization, canvas zoom, updated UI, bug fixes and performance improvements.
```
Comment (dreamofabear) [inline readme.txt]:
```suggestion
Automated video optimization, canvas zoom, updated UI, bug fixes and performance improvements.
```

Nit: "new" is implied.

**Review by dreamofabear — APPROVED** [review]

File: readme.txt (hunk 2 above)
Comment (swissspidy) [reply readme.txt]:
```suggestion
* New feature: automated video optimization during upload, enabling support for MOV files.
```

**Pull request merged**